lib/config.js

```diff
@@ -0,0 +1,16 @@
+var nconf = require('nconf');
+var dotenv = require('dotenv');
+
+dotenv.load();
+
+nconf.use('memory');
+nconf.env();
+
+nconf.defaults({
+  aws: {
```

Comment:
Why use the aws namespace? If you just make the key "AWS_ACCESS_KEY_ID" then it will be autoloaded from the environment?